pkg/controller/handlers/knowledgesource/knowledgesource.go

```diff
@@ -258,7 +258,7 @@ func (k *Handler) Sync(req router.Request, _ router.Response) error {
 
 	invokeOpts := invoke.SystemTaskOptions{
 		CredentialContextIDs: []string{credentialContextID},
-		Timeout:              1 * time.Hour,
+		Timeout:              2 * time.Hour,
```

Comment:
Increase the time since we want to increase the number of pages our scraper can scrape and sometimes it can take up more than 1 hour to finish.